src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,105 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.16.0.dev0 (3/29/2019)
+-----------------------
+
+New features
+~~~~~~~~~~~~
+
+* Add `./pants generate-pants-ini` for first time users to generate pants.ini with sensible defaults (#7448)
+  `PR #7448 <https://github.com/pantsbuild/pants/pull/7448>`_
+
+* Allow alternate nodejs binaries (#7405)
+  `PR #7405 <https://github.com/pantsbuild/pants/pull/7405>`_
+
+* Cache resolution of Node targets via fingerprinting lockfiles. (#7414)
```

Comment:
This is probably not a feature, because it's supposed to be invisible to the users (for now, they have not way of interacting with it).